egs2/TEMPLATE/asr1/asr.sh

```diff
@@ -766,6 +772,9 @@ if ! "${skip_data_prep}"; then
 
             _opts="--non_linguistic_symbols ${nlsyms_txt}"
 
+            if ${sot_asr} && [ "${token_type}" = char ]; then
```

Comment:
- The main difference between `sot_asr1` and `asr1`  is the data simulation process and the training data. If merging these two directories, we may need to introduce additional files (eg. `data/{train_sot,dev_sot,test_sot}`, `run_sot.sh`). I think it is ok to add some files but the results in `README.md` can not be compared with each other (due to the different training data, full-overlapped or partial-overlapped).
- I will add more descriptions about SOT
- "only char case?", in our experiments, SOT only converges well on char units, thus we only considered this case, I will check the compatibility with other units like BPE.

Reply:
OK, thanks.

Reply:
> only char case?
When we prepare the text, `<sc>` is inserted between speakers, with space on the left and right.
1) For the word case, `<sc>` is automatically regarded as a single word unit. So no special process for it.
2) For the BPE case, we use `_opts_spm+=" --user_defined_symbols=<sc>"` to take care of it. It will not be mixed in other BPE subwords.
3) For the char case, we need to add this special arguments to let the text processing know this is a single unit. It wouldn't be split into `<`, `s`, `c`, `>`.